smoke-test/src/test/java/org/opennms/smoketest/ClassificationRulePageIT.java

```diff
@@ -486,7 +486,9 @@ public void classifyAndWaitUntilAsserted(ClassificationRequestDTO classification
             execute(() -> findElementById("classification-submit")).click();
 
             await()
-                    .atMost(100, TimeUnit.SECONDS)
+                    // reloading the classification engine sometimes takes insanely long on CI
+                    // (this is further aggravated by the fact that one reload may be under way and another one may be pending)
```

Comment:
Would it be possible, then, to have a new reload cancel a previous one, other than (presumably) queue-ing after it?

Reply:
Good idea. I created an issue: https://issues.opennms.org/browse/NMS-13547

Reply:
I think you meant https://issues.opennms.org/browse/NMS-13580 but yup, that's good